### DISCUSSION TITLE
Gemini and Docx support! 🎉

### DISCUSSION THREAD

**Discussion by assafelovic** [assafelovic/gpt-researcher#409] [Announcements]
This release is largely thanks to @proy9714 for some high impact contributions!
1. Support for downloading research as docx file
2. Support for Gemini provider 
3. Improved report generation with hyperlinks
4. Support for Python3.10

Huge shoutout to @proy9714 👏

<hr /><em>This discussion was created from the release <a href='https://github.com/assafelovic/gpt-researcher/releases/tag/v.0.1.6'>Gemini and Docx support! 🎉</a>.</em>

**Comment by proy9714** [comment]
Such a wonderful mention....thank you so much 😄
